homework-submission/todo-app/src/Components/Task.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import '../App.css';
+
+class Task extends Component {
+    render() {
+        return (
+            <div className='Task'>
+                <div style={{ textDecoration: this.props.todo.done ? 'line-through' : 'none' }}>
+                    <li>{this.props.description}, {this.props.deadline} </li>
```

Comment:
This `li` tag is not in it's place, you can just remove it while you have a div can hold the text instead, or you can replace it with a `p` instead

Reply:
Hi Jawhar, thank you for your feedback, I have followed it. It was really helpful.

Regards,